yelp_package/dockerfiles/itest/marathon/Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN echo "debconf shared/accepted-oracle-license-v1-1 select true" | debconf-set
 RUN echo "debconf shared/accepted-oracle-license-v1-1 seen true" | debconf-set-selections
 RUN apt-get update && apt-get -y install lsb-release oracle-java8-installer
 
-RUN apt-get -y install libsasl2-modules marathon=0.11.1-1.0.432.ubuntu1404
+RUN apt-get -y install libsasl2-modules marathon=0.13.1-1.0.456.ubuntu1404
```

Comment:
Should be fine, this library is tested up to .14
https://github.com/thefactory/marathon-python/blob/master/.travis.yml